Source/Core/Common/GekkoDisassembler.cpp

```diff
@@ -459,14 +466,11 @@ std::string GekkoDisassembler::fd_ra_rb(u32 in, int mask)
   if (mask)
   {
     if (mask & 4)
-      result += fmt::format("f{},", PPCGETD(in));
```

Comment:
Feel free to correct me if I'm wrong as I don't know these instructions much.

Does your approach work when `mask & 4` is 0 and `mask & 2` is 1. Won't it produce a heading coma. If not, you should probably leave a comment stating why this approach is working.

Otherwise, what's wrong with the previous approach? If you're looking just to add a space character, can't you do it directly? Maybe using `fmt::join` would be cleaner.

Reply:
This confuses me, as it's only called by fdab and fdab only receives masks of 7 (all true). It could be some remnant of earlier code, but I wasn't sure I should change it completely. I can rewrite it and remove the mask if people are okay with it. Regardless, mask & 4 must always be true, as it's a float operation and that requires F{} register at the start.

Reply:
After thinking about it some more, there's no reason to keep the mask stuff. Removed the mask parameter from fdab instructions.